pkg/stockings/stock.go

```diff
@@ -0,0 +1,195 @@
+package stockings
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+/*
+	Could look to make this an interface as we move forward with different stock apis
+	ie make different structs for td, iex, rh apis and implement each of these methods for the
+	different apis via StockAPI interface.
+
+*/
+type IexStock struct {
+	Symbol        string  `json:"symbol"`
+	Name          string  `json:"companyName"`
+	LatestPrice   float32 `json:"latestPrice"`
```

Comment:
change this type to `decimal`: https://github.com/shopspring/decimal - if you rebase, you can see in `pkg/wardrobe/transfers.go` and `pkg/wardrobe/orders.go` for example on how I use this library